### PR TITLE
Fix whitespace trimming

### PR DESCRIPTION
**Pull request by cflynn07**

### CODE REVIEW

**Comment by cflynn07** [comment]
Just a few items to clean up in this PR, will fix shortly

File: index.js [hunk 1]
```diff
@@ -14,19 +14,6 @@ var streams = {
  * @return {DuplexStream} strim - stream whitespace trimmer
  */
 function createStrim () {
```
Comment (tjmehta) [inline index.js]: consider adding options; `{ blank: true, trailing: true }`

File: package.json [hunk 2]
```diff
@@ -4,7 +4,7 @@
   "description": "trim whitespace in streams",
   "main": "index.js",
   "scripts": {
-    "test": "lab -v -c",
+    "test": "lab -v -c -i 1",
```
Comment (tjmehta) [inline package.json]: remove -i 1

File: lib/remove-blank-lines.js [hunk 3]
```diff
@@ -1,36 +1,59 @@
...
+  var lastChunkLastLine = null;
+  var lastParsedChunk = null;
+  var atLeastOneLine = false;
+  function toString (chunk) {
+    return isString(chunk) ?
+      chunk : chunk.toString();
+  }
+  function write (chunk) {
+    var chunkString = toString(chunk);
+    var lines = chunkString.split(/\r\n|\n/);
```
Comment (tjmehta) [inline lib/remove-blank-lines.js]: might split on `\r\n`, but later on join by `\n`

Comment (tjmehta) [inline lib/remove-blank-lines.js]: ignoring this for now, as currently, it is not necessary for our use case.

**Pull request merged**